views/panel.php

```diff
@@ -20,6 +20,12 @@
 
   <title>Kirby Panel</title>
 
+  <script type="importmap">
+	{
+	  "imports": <?= json_encode($assets['import-maps'] ?? []) ?>
+	}
+	</script>
```

Comment:
`Todo:` check if Kirby itself could start using Vue via an import map in v5 already.

Reply:
https://github.com/getkirby/kirby/tree/v5/fix/panel-assets-import-map